<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestMode</class>
 <widget class="QWidget" name="TestMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>221</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="wordLabel">
      <property name="text">
       <string>Слово</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="answerInput"/>
    </item>
    <item>
     <widget class="QPushButton" name="submitButton">
      <property name="text">
       <string>Ответить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string>Результат</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="finishButton">
      <property name="text">
       <string>Завершить тест</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="progressLabel">
      <property name="text">
       <string>Прогресс</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
